forge/db/controllers/Device.js

```diff
@@ -75,11 +85,24 @@ module.exports = {
             return { name, value: value || '', platform: true } // add `platform` flag for UI
         }
         const result = []
+        let snapshotId
+        let snapshotName
+
+        if (device.ownerType === 'application') {
+            snapshotId = '0' // '0' is temporary value to indicate that the device should start in application mode with starter flows
+            snapshotName = 'None'
```

Comment:
I wonder if we should set them blank rather than these non-blank values. I appreciate why you set it to `0` for the internal usage between platform and agent, but in the flows, I feel like a blank value would be clearer so we don't have to explain it away otherwise.

Reply:
This is _very_ temporary Nick. 

Happy to use a name or something other than 'None'

Are you proposing using `Blank` or `Something-else` as the name?

Are you also proposing using something other than `0` for the ID?  I was torn between somthing like `DEFAULT` ot `APPLICATION-SNAPSHOT` but ultimately, the temporary nature of this, I went with '0'

Reply:
In terms of the env vars on the device, I was meaning literally blank - ''.

Reply:
hmmm. Would have to double check the logic across platform and device. 

For example, `'0' is used as a catch point in the device logic to know that A) the snapshot is _something_ and B) it is a _temporary/starter_snapshot for the device.  Existing logic for checking there IS a snapshot then works as is (because `.id` of '0' is _something_ and it signifies an application starter snapshot)

I do suspect this would cause some inconsistencies as it stands right now.

Happy to change - or move forward as is.  Please advise.

Reply:
I would have chosen to leave them blank as there is no running snapshot and I don't think the scenario of an existing device using logic based on this env var wanting to migrate to this new mode instantly (without snapshots in place) is really enough of a concern.

But I'm not going to push this.